continuous-integration/readme.md

```diff
@@ -26,29 +26,25 @@ These [principles](https://martinfowler.com/articles/continuousIntegration.html)
   - [Integration Validation](#integration-validation)
   - [Git Driven Workflow](#git-driven-workflow)
   - [Deliver Quickly and Daily](#deliver-quickly-and-daily)
-    - [Delivery Principles](#delivery-principles)
   - [Isolated Environments](#isolated-environments)
-    - [Environment Principles](#environment-principles)
   - [Developer Access to Latest Release Artifacts](#developer-access-to-latest-release-artifacts)
-    - [Artifact Principles](#artifact-principles)
   - [Integration Observability](#integration-observability)
-    - [CI Observability Principles](#ci-observability-principles)
   - [Resources](#resources)
 
 ## Goals
 
-Continuous integration automation is an integral part of the software development lifecycle intended to reduce build integration errors and maximize velocity across a dev crew. A robust build automation / validation pipeline will ultimately:
+Continuous integration automation is an integral part of the software development lifecycle intended to reduce build integration errors and maximize velocity across a dev crew.
 
-- Accelerate team velocity where build automation tool chains (Maven, MSBuild, etc) can run either locally or in a CI Server and continuously report errors.
+A robust build automation pipeline will:
+
+- Accelerate team velocity
 - Prevent integration problems
 - Avoid last minute chaos during release dates
-- Staging Builds
-- Enforces discipline of frequent automated testing
-- Quick feedback cycle for system-wide impact of local changes
-- Software Build and Deployment Separation
-- Measuring metrics around build failures / success(s)
+- Enforce discipline of test driven development
```

Comment:
Not sure I agree with this - doesn't this really depend on how developers choose to implement features with respect to the tests for those features?